_aggregations/metric/median-absolute-deviation.md

```diff
@@ -9,70 +9,37 @@ redirect_from:
 
 # Median absolute deviation aggregations
 
-The `median_absolute_deviation` metric is a single-value metric aggregation that returns a median absolute deviation field. Median absolute deviation is a statistical measure of data variability. Because the median absolute deviation measures dispersion from the median, it provides a more robust measure of variability that is less affected by outliers in a dataset. 
+The `median_absolute_deviation` metric is a single-value metric. Median absolute deviation is a variability metric that measures dispersion from the median.
 
-Median absolute deviation is calculated as follows:<br>
-median_absolute_deviation = median(|X<sub>i</sub> - Median(X<sub>i</sub>)|)
+Median absolute deviation is less affected by outliers than standard deviation, which relies on squared error terms, and is useful for describing data that is not normally distributed.
 
-The following example calculates the median absolute deviation of the `DistanceMiles` field in the sample dataset `opensearch_dashboards_sample_data_flights`:
+Median absolute deviation is computed as follows:<br/>
+median_absolute_deviation = Median( | X<sub>i</sub> - Median(X<sub>i</sub>) | )
 
+OpenSearch estimates `median_absolute_deviation`, rather than calculating it directly, due to memory limitations. This estimation is computationally expensive. You can adjust the trade-off between estimation accuracy and performance; see [Adjusting estimation accuracy](https://github.com/opensearch-project/documentation-website/pull/9453/files#adjusting-estimation-accuracy) for details.
```

Comment:
```suggestion
OpenSearch estimates `median_absolute_deviation`, rather than calculating it directly, because ofmemory limitations. This estimation is computationally expensive. You can adjust the trade-off between estimation accuracy and performance. For more information, see [Adjusting estimation accuracy](https://github.com/opensearch-project/documentation-website/pull/9453/files#adjusting-estimation-accuracy).
```

Reply:
```suggestion
OpenSearch estimates `median_absolute_deviation`, rather than calculating it directly, because of memory limitations. This estimation is computationally expensive. You can adjust the trade-off between estimation accuracy and performance. For more information, see [Adjusting estimation accuracy](https://github.com/opensearch-project/documentation-website/pull/9453/files#adjusting-estimation-accuracy).
```